client/src/components/AttractionShowPage.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+
+import ReviewTile from "./ReviewTile.js";
+
+import translateServerErrors from "../services/translateServerErrors.js";
+
+const AttractionShowPage = (props) => {
+  //debugger
+  const [attraction, setAttraction] = useState({
+    attractions: [],
```

Comment:
I don't think we want an array of `attractions` here on the show page, but instead the one attraction object with it's reviews